scripts/run-registry.ts

```diff
@@ -124,6 +145,19 @@ const publish = (packages: { name: string; location: string }[], url: string) =>
   );
 };
 
+// const addUser = (url: string) =>
+//   new Promise((res, rej) => {
+//     logger.log(`👤 add temp user to verdaccio`);
+
+//     exec(`npx npm-cli-adduser -r "${url}" -a -u user -p password -e user@example.com`, (e) => {
+//       if (e) {
+//         rej(e);
+//       } else {
+//         res();
+//       }
+//     });
+//   });
```

Comment:
I had to run this to get it to work locally, so I added this bit of code back. But it seems the CI doesn't need it, so commented it again.